include/swift/AST/Module.h

```diff
@@ -1117,20 +1118,23 @@ class SourceFile final : public FileUnit {
   /// Set the root refinement context for the file.
   void setTypeRefinementContext(TypeRefinementContext *TRC);
 
+  void initInterfaceHashIfEmpty() {
```

Comment:
This doesn't seem right to me. I'd expect something like `enableInterfaceHash()`, which would assert if it were already enabled.

Reply:
That makes sense.